tests/snapshots/tests.contrib.botocore.test_bedrock.test_converse_stream.json

```diff
@@ -0,0 +1,35 @@
+[[
+  {
+    "name": "bedrock-runtime.command",
+    "service": "aws.bedrock-runtime",
+    "resource": "ConverseStream",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "",
```

Comment:
Note: I'm asking around to see if there's a preference for setting span type as empty or if this should just be `custom`.

Reply:
Worth following something like this in the future then to be consistent: https://docs.datadoghq.com/opentelemetry/mapping/semantic_mapping/?tab=datadogexporter#map-opentelemetry-span-attribute-to-datadog-span-type